app/views/node_info/_statistics.haml

```diff
@@ -0,0 +1,21 @@
+-#   Copyright (c) 2010-2011, Diaspora Inc.  This file is
```

Comment:
Right, I just moved the file, I don't include it to new files for the time being.